config.default

```diff
@@ -47,6 +47,10 @@ seaIceStreamsFileName = streams.cice
 # names of ocean and sea ice meshes (e.g. EC60to30, QU240, RRS30to10, etc.)
 mpasMeshName = mesh
 
+# this is the maximum number of files, beyond which xarray will close files 
+# after data has been read to prevent errors related to too many open files.
+autocloseFileCount = 1024
```

Comment:
I don't think we should specify this necessarily.  It may be better to use something like `ulimit -a | grep open | awk '{print $4}'` and a heuristic to determine if we need `autoclose=True`.